extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -431,22 +434,50 @@
 			}
 		};
 
-		if (_t.updateInterval === 0) {
-			return;
-		}
-		// Limit the refresh interval to 100 ms
-		if (_t.updateInterval < 100) {
-			_t.updateInterval = 100;
-		}
+		_t.setVisible = function(state) {
+			if (state) {
+				_t.formRow.classList.remove(o.formRowHidden);
+				_t.activateRefresh();
+			} else {
+				_t.formRow.classList.add(o.formRowHidden);
+				_t.disactivateRefresh();
+			}
+
+			_t.visible = state;
+		};
+
+		_t.disactivateRefresh = function() {
+			if (interval !== undefined) {
+				clearInterval(interval);
+				interval = undefined;
```

Comment:
JS type are weird but I think here `null` is more suitable as it's used in the rest of the code

Reply:
Ok I will change that.